test/user.js

```diff
@@ -229,3 +233,125 @@ describe('/users POST', () => {
       });
   });
 });
+
+describe('/users/sessions POST', () => {
+  it('should successfully log in a user', done => {
+    signUpTestUser().then(res => {
+      // user should been successfully created
+      res.should.have.status(201);
+      User.count({ where: { email: testEmail } })
+        .then(count => {
+          should.equal(count, 1);
+        })
+        .then(() => {
+          chai
+            .request(server)
+            .post('/users/sessions')
+            .set('content-type', 'application/json')
+            .send({ email: testEmail, password: testPassword })
+            .then(r => {
+              should.exist(r.body.token);
+              should.not.equal(r.body.token, '', 'token shouldnt be empty');
```

Comment:
In general,we verify the result is equal , no if the result is not equal . Maybe decode token and verify if exist mail and check if is the one created previously